data/addFieldsToCollection.js

```diff
@@ -88,6 +109,77 @@ async function setLevelInCourses() {
   });
 }
 
+/* 
+  Script to calculate the post reqs for each sis course.
+*/
+async function setPostReqsInSISCourses() {
+  await db.connect();
+  var counter = 0;
+  SISCV.find({ version: { $elemMatch: { postReq: { $size: 0 } } } }).then(
+    async (res) => {
+      for (let course of res) {
+        let postReqs = [];
+        await SISCV.find({
+          versions: {
+            $elemMatch: {
+              preReq: { $elemMatch: { Expression: { $regex: course.number } } },
+            },
+          },
+        }).then(async (res2) => {
+          for (let matchedCourse of res2) {
+            for (let preReq of matchedCourse.versions[0].preReq) {
+              if (preReq.IsNegative === 'N') {
```

Comment:
comments about what IsNegative is would be nice!

Reply:
Will do, not entirely sure myself, but based on the database, it seems to be courses you cannot take at the same time as this(like you can't do intro prob and prob/stats together and those type of info/conditions are stored as a pre-req in our database). Will double check and add comment.